.github/workflows/stable-release.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       - run: |
           echo "RELEASE_VERSION=$(./scripts/version-from-git-tag.sh)" >> $GITHUB_ENV
       - run: echo "Publishing version $RELEASE_VERSION"
+      - run: git tag -a "RELEASE_VERSION" -m "RELEASE_VERSION" && git push origin "$RELEASE_VERSION"
```

Comment:
I don't think it will work on CI

Reply:
you are right. let's stick to the current approach then